native/sapp-windows/src/lib.rs

```diff
@@ -671,11 +737,43 @@ unsafe extern "system" fn win32_wndproc(
                         sapp_mousebutton_SAPP_MOUSEBUTTON_INVALID,
                     );
                 }
+
                 _sapp_win32_mouse_event(
                     sapp_event_type_SAPP_EVENTTYPE_MOUSE_MOVE,
                     sapp_mousebutton_SAPP_MOUSEBUTTON_INVALID,
                 );
             }
+
+            WM_MOVE if _sapp.cursor_grabbed => {
+                update_clip_rect(hWnd);
+            }
+
+            WM_INPUT => {
+                let mut data: RAWINPUT = std::mem::zeroed();
+                let mut size = std::mem::size_of::<RAWINPUT>();
+                let get_succeed = GetRawInputData(
+                    lParam as _,
+                    RID_INPUT,
+                    &mut data as *mut _ as _,
+                    &mut size as *mut _ as _,
+                    std::mem::size_of::<RAWINPUTHEADER>() as _,
+                );
+                if get_succeed as i32 == -1 {
+                    panic!("failed to retrieve raw input data");
+                }
+
+                if data.data.mouse().usFlags & MOUSE_MOVE_ABSOLUTE == 1 {
+                    panic!("grr absolute coords");
```

Comment:
shouldnt we just ignore the ABSOLUTE part of the event if it happens to be absolute? (I am not sure how it works, so maybe panic is actually the right thing to do)

Reply:
yeah, I'm not sure. glfw has a bunch of code to track the last known position and still provide deltas even if the hardware uses absolute coordinates, but I heard hardware very rarely if ever gives off these events. sure enough, I tested all of the mice I had and they all gave off relative mouse coords. I think support for absolute coords should only be added if someone is actually running into this panic, because I think ABSOLUTE is just legacy at this point.

Reply:
Got it, sounds reasonable!